dev-docs/Development.md

```diff
@@ -15,6 +15,51 @@ pre-commit install # install the pre commits hooks
 pre-commit run --all-files --verbose # run it!
 ```
 
+
+### Update default configurations
+
+If you need to change the default values in the configuration(`ES_HOST`, `ES_PORT`, `ES_USERNAME`, `ES_PASSWORD`), you
+can also create the deployment file yourself:
+
+Self-Managed Kubernetes
+```zsh
+just create-vanilla-deployment-file
```

Comment:
@oren-zohar I'm retrieving the following error:
kustomize build deploy/kustomize/overlays/cloudbeat-vanilla --output deploy/k8s/cloudbeat-ds.yaml
Error: loading KV pairs: file sources: [./ca-cert.pem]: evalsymlink failure on '/cloudbeat/deploy/kustomize/overlays/cloudbeat-vanilla/ca-cert.pem' : lstat /cloudbeat/deploy/kustomize/overlays/cloudbeat-vanilla/ca-cert.pem: no such file or directory
error: Recipe `create-vanilla-deployment-file` failed on line 24 with exit code 1

Reply:
original command shoul've had   
```zsh
cp {{env_var('ELASTIC_PACKAGE_CA_CERT')}} {{kustomizeVanillaOverlay}}
```
adding